wagtailio/core/models/pages.py

```diff
@@ -22,17 +28,57 @@ class HomePage(Page, SocialMediaMixin, CrossPageMixin):
         "packages.PackagesPage",
         "services.ServicesPage",
     ]
-    content_panels = Page.content_panels + [StreamFieldPanel("body")]
+
+    code_snippet = models.CharField(
```

Comment:
issue: the designs contain multiple lines. as it stands the editor is presented with a regular text input so multiple lines are awkward at best.

I would make it optional to add the code snippet, cta and video

Reply:
Thanks, the  code snippet, call to action and video are optional, and a TextField used instead of CharField for the code snippet